docs/nodes/full-consensus-node.mdx

```diff
@@ -184,30 +194,30 @@ copied from.
 
 Run the following command to quick-sync from a snapshot for `mocha`:
 
-```sh
-cd $HOME
-rm -rf ~/.celestia-app/data
-mkdir -p ~/.celestia-app/data
-SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \
-    egrep -o ">mocha-3.*tar" | tr -d ">")
-wget -O - https://snaps.qubelabs.io/celestia/${SNAP_NAME} | tar xf - \
-    -C ~/.celestia-app/data/
-```
+<pre><code>
+cd $HOME<br/>
+rm -rf ~/.celestia-app/data<br/>
+mkdir -p ~/.celestia-app/data<br/>
+SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \<br/>
+    egrep -o ">{constants.mochaChainId}.*tar" | tr -d ">")<br/>
+wget -O - https://snaps.qubelabs.io/celestia/$&#123;SNAP_NAME&#125; | tar xf - \<br/>
+    -C ~/.celestia-app/data/<br/>
+</code></pre>
 
 </TabItem>
-<TabItem value="blockspacerace" label="Blockspace Race">
+<TabItem value="arabica" label="Arabica">
 
-Run the following command to quick-sync from a snapshot for `blockspacerace`:
+Run the following command to quick-sync from a snapshot for `arabica-9`:
 
-```sh
-cd $HOME
-rm -rf ~/.celestia-app/data
-mkdir -p ~/.celestia-app/data
-SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \
-    egrep -o ">blockspacerace.*tar" | tr -d ">")
-wget -O - https://snaps.qubelabs.io/celestia/${SNAP_NAME} | tar xf - \
-    -C ~/.celestia-app/data/
-```
+<pre><code>
+cd $HOME<br/>
+rm -rf ~/.celestia-app/data<br/>
+mkdir -p ~/.celestia-app/data<br/>
+SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \<br/>
+    egrep -o ">{constants.arabicaChainId}.*tar" | tr -d ">")<br/>
```

Comment:
@mindstyle85 - do you all have a snapshot hosted for this? if not, @Bidon15 we may need to run our own and i will edit this accordingly

Reply:
> @qubelabsio - do you all have a snapshot hosted for this? if not, @Bidon15 we may need to run our own and i will edit this accordingly

@jcstein both are now maintained at https://snaps.qubelabs.io/celestia/, so just use the `mocha-3` and `arabica-9` tags.

Reply:
amazing, thank you!